<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsTab</class>
 <widget class="QWidget" name="OptionsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>384</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="vboxMain">
   <item>
    <widget class="QGroupBox" name="grpDownloads">
     <property name="title">
      <string>&amp;Downloads</string>
     </property>
     <layout class="QVBoxLayout" name="vboxDownloads">
      <item>
       <widget class="QCheckBox" name="chkExtImgDownloadEnabled">
        <property name="text">
         <string>Enable external image downloads.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkUseIntIconForSmallSizes">
        <property name="text">
         <string>Always use the internal icon (if present) for small sizes.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkDownloadHighResScans">
        <property name="text">
         <string>Download high-resolution scans if viewing large thumbnails.
This may increase bandwidth usage.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkStoreFileOriginInfo">
        <property name="text">
         <string>Store cached file origin information using extended attributes.
This helps to identify where cached files were downloaded from.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hboxGameTDBPAL">
        <item>
         <widget class="QLabel" name="lblGameTDBPAL">
          <property name="text">
           <string>Language for PAL titles on GameTDB:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LanguageComboBox" name="cboGameTDBPAL"/>
        </item>
        <item>
         <spacer name="hspcGameTDBPAL">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpOptions">
     <property name="title">
      <string>&amp;Options</string>
     </property>
     <layout class="QVBoxLayout" name="vboxOptions">
      <item>
       <widget class="QCheckBox" name="chkShowDangerousPermissionsOverlayIcon">
        <property name="text">
         <string>Show a security overlay icon for ROM images with
&quot;dangerous&quot; permissions.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkEnableThumbnailOnNetworkFS">
        <property name="text">
         <string>Enable thumbnailing and metadata extraction on network
file systems. This may slow down file browsing.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vspcDownloads">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>197</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>LanguageComboBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkExtImgDownloadEnabled</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseIntIconForSmallSizes</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>26</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDownloadHighResScans</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkShowDangerousPermissionsOverlayIcon</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEnableThumbnailOnNetworkFS</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStoreFileOriginInfo</sender>
   <signal>clicked()</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboGameTDBPAL</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>OptionsTab</receiver>
   <slot>optionChanged_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>modified()</signal>
  <slot>reset()</slot>
  <slot>loadDefaults()</slot>
  <slot>save()</slot>
  <slot>optionChanged_slot()</slot>
 </slots>
</ui>
